test.py

```diff
@@ -1,124 +1,527 @@
-
-add_test('test_new_pr.json', {'new_contributor': True},
-         {'labels': ['S-awaiting-review'], 'comments': 1})
-
-add_test('test_ignored_action.json', {}, {})
-
-add_test('test_synchronize.json', {'labels': ['S-needs-code-changes', 'S-tests-failed', 'S-awaiting-merge']},
-         {'labels': ['S-awaiting-review']})
-
-add_test('test_comment.json', {}, {'assignee': 'jdm'})
-
-add_test('test_merge_approved.json', {'labels': ['S-needs-code-changes', 'S-needs-rebase',
-                                                 'S-tests-failed', 'S-needs-squash',
-                                                 'S-awaiting-review']}, {'labels': ['S-awaiting-merge']})
-
-add_test('test_merge_conflict.json', {'labels': ['S-awaiting-merge']},
-         {'labels': ['S-needs-rebase']})
-
-add_test('test_tests_failed.json', {'labels': ['S-awaiting-merge']},
-         {'labels': ['S-tests-failed']})
-
-add_test('test_post_retry.json', {'labels': ['S-tests-failed']},
-         {'labels': ['S-awaiting-merge']})
-
-add_test('test_post_retry.json', {'labels': ['S-awaiting-merge']},
-         {'labels': ['S-awaiting-merge']})
-
-run_tests(tests)
+error_sample = [ 
+    {
+        "body": "use statement is not in alphabetical order\nexpected: dom::bindings::codegen::Bindings::EventHandlerBinding::EventHandlerNonNull\nfound: dom::bindings::conversions::get_dom_class", 
+        "position": 7, 
+        "path": "./components/script/dom/eventtarget.rs"
+    }, 
+    {
+        "body": "use statement is not in alphabetical order\nexpected: dom::bindings::codegen::Bindings::EventListenerBinding::EventListener\nfound: dom::bindings::codegen::Bindings::EventHandlerBinding::EventHandlerNonNull", 
+        "position": 8, 
+        "path": "./components/script/dom/eventtarget.rs"
+    }, 
+    {
+        "body": "use statement is not in alphabetical order\nexpected: dom::bindings::codegen::Bindings::EventTargetBinding::EventTargetMethods\nfound: dom::bindings::codegen::Bindings::EventListenerBinding::EventListener", 
+        "position": 9, 
+        "path": "./components/script/dom/eventtarget.rs"
+    }, 
+    {
+        "body": "use statement is not in alphabetical order\nexpected: dom::bindings::conversions::get_dom_class\nfound: dom::bindings::codegen::Bindings::EventTargetBinding::EventTargetMethods", 
+        "position": 10, 
+        "path": "./components/script/dom/eventtarget.rs"
+    }, 
+    {
+        "body": "use statement is not in alphabetical order\nexpected: dom::browsercontext\nfound: dom::eventtarget::EventTargetTypeId", 
+        "position": 17, 
+        "path": "./components/script/dom/bindings/utils.rs"
+    }, 
+    {
+        "body": "use statement is not in alphabetical order\nexpected: dom::eventtarget::EventTargetTypeId\nfound: dom::browsercontext", 
+        "position": 18, 
+        "path": "./components/script/dom/bindings/utils.rs"
+    }
+]
+
+
+class TestAPIProvider(unittest.TestCase):
```

Comment:
I don't think this test case is doing anything besides improving our code coverage.